std/traits.d

```diff
@@ -3804,6 +3811,32 @@ template hasStaticMember(T, string member)
     static assert( hasStaticMember!(C, "iosf"));
     static assert(!hasStaticMember!(C, "p"));
     static assert( hasStaticMember!(C, "sp"));
+
+    alias P = S*;
+    static assert(!hasStaticMember!(P, "na"));
+    static assert(!hasStaticMember!(P, "X"));
+    static assert(!hasStaticMember!(P, "Y"));
+    static assert(!hasStaticMember!(P, "Y.i"));
+    static assert(!hasStaticMember!(P, "S"));
+    static assert(!hasStaticMember!(P, "C"));
+    static assert( hasStaticMember!(P, "sx"));
+    static assert( hasStaticMember!(P, "gx"));
+    static assert(!hasStaticMember!(P, "y"));
+    static assert( hasStaticMember!(P, "sy"));
+    static assert( hasStaticMember!(P, "f"));
+    static assert( hasStaticMember!(P, "f2"));
+    static assert(!hasStaticMember!(P, "dm"));
+    static assert( hasStaticMember!(P, "sd"));
+    static assert(!hasStaticMember!(P, "g"));
+    static assert( hasStaticMember!(P, "fp"));
+    static assert( hasStaticMember!(P, "gfp"));
+    static assert(!hasStaticMember!(P, "fpm"));
+    static assert(!hasStaticMember!(P, "m"));
+    static assert(!hasStaticMember!(P, "m2"));
+    static assert(!hasStaticMember!(P, "iom"));
+    static assert( hasStaticMember!(P, "iosf"));
+    static assert(!hasStaticMember!(P, "p"));
+    static assert( hasStaticMember!(P, "sp"));
```

Comment:
Maybe all these tests are unnecessary to just see if T* doesn't fail?

Reply:
No more tests are almost always helpful.